jenkins-jobs/scripts/release-pipeline.groovy

```diff
@@ -0,0 +1,376 @@
+import groovy.json.*
+import java.util.stream.*
+
+if (!RELEASE_VERSION || !DEVELOPMENT_VERSION) {
+    error 'Input parameters not provided'
```

Comment:
Should the branch version be required? Would be nice to be able to release a version using a branch other than master.